docs/server/installation.rst

```diff
@@ -7,71 +7,18 @@ These instructions will guide you through installing Pootle and its
 requirements in a virtual environment.
 
 If you only want to have a sneak peek of Pootle then the default configuration
-and the built-in server will suffice. But in case you want to deploy a real
-world server, :ref:`installing optional packages
-<optimization#optional_software>`, using a real database and :doc:`a proper web
-server <web>` is highly recommended.
+and the built-in server will suffice.
 
-The easiest way to test and install Pootle is by using pip, however,
-installating it straight from git sources is another viable approach.
+For a production deployment we recommend that you set up the following:
 
+- :ref:`install optional packages<optimization#optional_software>`
+- use either a :ref:`MySQL <mysql_installation>`
+  or :ref:`PostgreSQL <postgresql_installation>` database.
+- :doc:`make use of a front-end web server <web>`
 
-.. _installation#requirements:
 
-Requirements
-------------
-
-
-.. _installation#hardware_requirements:
-
-Hardware Requirements
-^^^^^^^^^^^^^^^^^^^^^
-
-Your Pootle installation will need to be flexible enough to handle the
-translation load. The recommended hardware depends highly on the performance you
-expect, the number of users you want to support, and the number and size of the
-files you want to host.
-
-Whatever hardware you have, you will still benefit from performance improvements
-if you can :doc:`optimize your system <optimization>`.
-
-Your disk space should always be enough to store your files and your Pootle
-database, with some extra space available.
-
-
-.. _installation#system_requirements:
-
-System Requirements
-^^^^^^^^^^^^^^^^^^^
-
-To run Pootle you need a computer running:
-
-- Linux
-- Mac OS X
-
-Or, any other Unix-like system.
-
-.. note:: Pootle will not run on Windows since it uses RQ, whose workers cannot
-   run on `Windows <http://python-rq.org/docs/>`_.
-
-   Some developers do develop on Windows so these problems can be worked around
-   for some of the development tasks.
-
-   Pootle should be able to run on any system that implements ``fork()``.
-
-
-Software requirements
-^^^^^^^^^^^^^^^^^^^^^
-
-**Python 2.7 is required**. 2.6 won't work, and 3.x is not supported.
-
-You will also need the following system services for a working Pootle:
-
-- Redis - all caching and managing of workers uses Redis
-- Database - MySQL or PostgreSQL (although for testing SQLite is fine)
-- Elasticsearch (optional) - for Local Translation Memory
-
-These are preferably installed from system packages.
+.. note:: Before installing please ensure that you have all the
+   :ref:`necessary requirements <requirements>` and have a running Redis server.
```

Comment:
Having a running Redis server is a *necessary requirement* already. Any reasons to specifically mention it here?